examples/chart/teleport-kube-agent/templates/updater/deployment.yaml

```diff
@@ -71,7 +71,7 @@ spec:
     would be a breaking change when the teleport proxy starts override the explicitly set RFD-109 version server */ -}}
   {{- if and $updater.proxyAddr (not $versionServerOverride)}}
           - "--proxy-address={{ $updater.proxyAddr }}"
-          - "--update-group={{ default $updater.releaseChannel $updater.group }}"
```

Comment:
this looks breaking and should at least be mentionned in the changelog, we might also have to edit the comments in values.yaml